### PR TITLE
[IMP] hr_timesheet: Kanban view revamp

### PR DESCRIPTION
**Pull request by ppr-odoo**
-Kanban view revamp:
    -On the left: display the picture of the employee
    -In the middle: the name of the project and below, the name of the task
    -On the right: the duration with - / + buttons
-When you click on the + and - buttons, the time will change according
to the rounding of the timesheet setting.
-When the mouse hovers over the duration, the play button will appear and then
click on the play button, the timer will start, and the stop button will appear.

Task-ID: #2264613

### CODE REVIEW

File: addons/web/static/src/js/fields/basic_fields.js [hunk 1]
```diff
@@ -1299,7 +1299,7 @@ var FieldFloatToggle = AbstractField.extend({
      * @param {OdooEvent} ev
      */
     _onClick: function(ev) {
-        if (this.mode === 'edit') {
+        if (this.mode === 'edit' || this.nodeOptions.force_button) {
             ev.stopPropagation(); // only stop propagation in edit mode
```
Comment (arj-odoo) [inline addons/web/static/src/js/fields/basic_fields.js:1303]: You should adapt the comment if your condition is modified.

Comment (ppr-odoo) [reply addons/web/static/src/js/fields/basic_fields.js:1304]: @arj-odoo Done

Comment (arj-odoo) [reply addons/web/static/src/js/fields/basic_fields.js:1304]: you can delete the comment '// only stop propagation in edit mode'. it is not useful anymore.

Comment (ppr-odoo) [reply addons/web/static/src/js/fields/basic_fields.js]: @arj-odoo done

**Comment by tivisse** [comment]
@robodoo r+

**Comment by tivisse** [comment]
@rebase-ff

**Pull request closed without merging**